sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -108,11 +108,14 @@ export class Recorder {
    * addSanitizers adds the sanitizers for the current recording which will be applied on it before being saved.
    *
    * Takes SanitizerOptions as the input, passes on to the proxy-tool.
+   *
+   * Applied in both record and playback modes.
+   *
+   * (If you want this to be applied only in a specific mode(either "record" or "playback"), check the mode yourself and add the call accordingly.)
    */
   async addSanitizers(options: SanitizerOptions): Promise<void> {
-    // If check needed because we only sanitize when the recording is being generated, and we need a recording to apply the sanitizers on.
     if (
-      isRecordMode() &&
+      !isLiveMode() &&
```

Comment:
I get why we want to add support for this, but I don't know if I'm a fan of the two ways of adding sanitizers having different behavior. Can we do something like adding an extra (optional) option to each one to specify which modes we want to add the sanitizers to so that it's consistent?

Reply:
Hmmm, or leave the start call as is, and add an optional option to the `addSanitizers`?
By default, it only applies to "record" mode so that it is consistent with start, if the optional option is specified, we'll add it to playback as well.

I'm thinking this might be a better option since I don't see anyone using the sanitizers in playback, it'll just be more unnecessary calls to the service.

Reply:
Yeah, that sounds good. I suppose we don't need to both have them support it, as long as it's obvious when we're specifying that the sanitizer should be added in playback.